ui-tests/src/test/java/org/openmrs/reference/MarkPatientDeceasedTest.java

```diff
@@ -0,0 +1,50 @@
+package org.openmrs.reference;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.AdministrationPage;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.MarkPatientDeceasedPage;
+import org.openmrs.uitestframework.test.TestData;
+
+public class MarkPatientDeceasedTest extends ReferenceApplicationTestBase {
+	
+	private TestData.PatientInfo patient;
+	  
+	@Before
+    public void setUp() throws Exception {
+        patient = createTestPatient(); 
+        createTestVisit();
+        
+    }
+	
```

Comment:
Remove redundant lines here

Reply:
Thank you so much @sherrif10 let me work on them